app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadController.java

```diff
@@ -260,6 +256,21 @@ void microarrayExperimentDownload(String experimentAccession,
                                         experiment.getAccession(), ExperimentFileType.SUMMARY_PDF));
                         break;
 
+                    case PROTEOMICS_DIFFERENTIAL:
+                        paths.add(experimentFileLocationService.getFilePath(
+                                experiment.getAccession(), ExperimentFileType.CONDENSE_SDRF))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.CONFIGURATION))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.PROTEOMICS_RAW_QUANT))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.IDF))
+                                .add(experimentFileLocationService.getFilePath(
+                                        experiment.getAccession(), ExperimentFileType.RNASEQ_D_ANALYTICS))
```

Comment:
Isn't there a specific ExperimentFileType for the proteomics differential analytics file instead of reusing the RNA-Seq one? This might bite back later on and it looks a bit confusing to new developers.

Reply:
Or is it a complication to have one?

Reply:
We don't have the specific for that. We could have another story to add it if you think.

Reply:
I would prefer if we do add one, regardless of whether this will require another story, but on this same PR. Otherwise leaving this untidy will lead to errors in the future (maybe someone changing the RNA differential file pattern without realising that affects proteomics, etc).